Translations/fr.po

```diff
@@ -8119,9 +8124,9 @@ msgid ""
 "dead! Dead! Do you hear me? They just keep falling and falling... their "
 "blood spilling out all over the floor... all his fault..."
 msgstr ""
-"Ils poignardent, puis mordent, puis ils sont tout autour de vous. Menteur! "
-"MENTEUR! Ils sont tous morts ! Morts! Vous m'entendez? Ils n'arrêtent pas de "
-"tomber et de tomber... leur sang recouvre le sol... tout est sa faute..."
+"Ils plantent, puis mordent, puis ils sont tout autour de vous. Menteur ! "
```

Comment:
I prefere "poignardent" to "plantent". Or may be "frappent" ?

Reply:
I was trying to find a basic, instinctive word that could suit a drunkard’s mouth: I was feeling "poignarder" may be too elaborate for the state he’s in. Maybe "frapper" wourd be a better compromise.

Reply:
Or maybe "piquer"…?